packages/syncapi/src/customSyncStorage.ts

```diff
@@ -0,0 +1,34 @@
+import { HasId, Store } from '@raincatcher/store';
+
+
+class CustomSyncStorage<T extends HasId>{
+    public store: Store<T>;
+
+    public constructor(store: Store<T>) {
+        this.store = store;
+    }
+
+    public function dataListHandler(datasetId: any, queryParams: any, metadata: any, cb: any) {
+        
+    }
+    public function dataCreateHandler(datasetId: any, queryParams: any, metadata: any, cb: any) {
+
+    }
+
+    public function dataSaveHandler(datasetId: any, queryParams: any, metadata: any, cb: any) {
+
+    }
+
+    public function dataGetHandler(datasetId: any, queryParams: any, metadata: any, cb: any) {
+
+    }
+
+    public function dataDeleteHandler(datasetId: any, queryParams: any, metadata: any, cb: any) {
+
+    }
```

Comment:
A lot of `any`s here for now, but I think that's provisional until we use this and map closer to what sync feeds these functions.